README.md

```diff
@@ -1,51 +1,43 @@
-# Online store PWA sample.
+# Online store PWA sample
 
-This sample will attempt to demonstrate some best practices on the web, taking
-into account the specific requirements of an e-commerce site. It will also
-demonstrate some useful features enabled by new technologies and APIs on the
-web.
+This sample demonstrates best practice for e-commerce websites.
```

Comment:
I think the previous one was more descriptive.

Reply:
Fair point, though I think the description needs to make it immediately obvious what the repo is for. 

'demonstrate some best practices on the web' is a bit vague — 'e-commerce' is a bit buried :).

Reply:
Well "best practice" can mean anything - the right usage of colors? Theming? UX or design challenges? I like that we were explicit it is about new features and APIs on the web and web technologies. 

Maybe:
"This sample demonstrates some best practices on the web for e-commerce websites. It also
demonstrates some useful features enabled by new technologies and APIs on the
web."

Reply:
That sounds good — tweaked accordingly.